packages/core/src/web/app/components/dialogs/SocialMediaModal.tsx

```diff
@@ -27,60 +29,56 @@ const Item = ({ item }: { item: TItem }) => (
 );
 
 interface Props {
+  autoPopup?: boolean;
   onClose: () => void;
 }
 
-const SocialMediaModal = ({ onClose }: Props): React.JSX.Element => {
+const SocialMediaModal = ({ autoPopup = false, onClose: closeModal }: Props): React.JSX.Element => {
   const {
+    alert: tAlert,
     social_media: t,
     topbar: { menu: tMenu },
   } = useI18n();
+  const [dontShow, setDontShow] = useState(false);
```

Comment:
I think maybe name this to dontShowAgain would be better because `dontShow` is a little bit comfusing